CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is loosely based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
+## Unreleased
+
+#### Fixed
+
+* Added Hashset of active invocations to SetupCollection to only call MarkOverridden when a duplicated expectation is added (@CeesKaas, #1110)
```

Comment:
Most Moq users will probably have no idea what exactly this means, as you focus on Moq implementation details instead of what it means from a user's perspective. May I suggest that you rephrase this somewhere along these lines:

```suggestion
* Performance regression: Adding setups to a mock becomes slower with each setup (@CeesKaas, #1110)
```